production.ini.in

```diff
@@ -38,6 +38,7 @@ sqlalchemy.lubis.url = postgresql://${vars:dbhost}:${vars:dbport}/lubis
 mako.directories = chsdi:templates
 
 data_path = /var/local/cartoweb/
+zadara_downloads = /var/local/cartoweb/downloads/
```

Comment:
Anyhow, this variable should be put into the vars section of the buildout config and used here as ${vars:zadara_dir}, then use the same variable in the apache configuration (during mako creation).